framework/src/st_things/st_things_logger.c

```diff
@@ -45,7 +45,7 @@ void st_things_log_init()
 void st_things_log_shutdown()
 {
 	/* Free the memory allocated for p_version */
-	free(p_version);
+	util_free(p_version);
```

Comment:
not ```things_free```? What is difference between things and util? Isn't ```util``` a part of st_things?

Reply:
util_ is used in st_things SDK level
things_ is used in st_things stack inside.

Reply:
1. Are SDK and stack separated by folder? or Is it mentioned on file the difference?
2. Anyway SDK is a part of st_things as well. Is there any chance getting memory usages separately?

Reply:
1. yes. its separated by folder 
 - sdk : framework/st_things 
 -stack : framework/st_things/things_stack
2. Yes. So i call things_malloc(in util_malloc)
However, it can be separated later to distinguish it from the SDK (APP).

Reply:
let's apply with unified rule with next commit